tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -519,6 +519,25 @@ public function testRenameColumn()
         $this->assertTrue($this->adapter->hasColumn('t', 'column2'));
     }
 
+    public function testRenameColumnPreserveComment()
+    {
+        $table = new \Phinx\Db\Table('t', [], $this->adapter);
```

Comment:
Ideally, the FQCN is inside the use statement here.

Reply:
`$table = new Table('t', [], $this->adapter);`

Reply:
All other tests have it in `$table = new \Phinx\Db\Table('t', [], $this->adapter);` format. Should I change this one?

Reply:
It’s ok, no need to change the current style right now.

Reply:
They should all be fixed at some point.
[This tool](https://github.com/spryker/code-sniffer) can help doing at :)